pkg/flowaggregator/flowaggregator_test.go

```diff
@@ -441,6 +441,7 @@ func TestFlowAggregator_updateFlowAggregator(t *testing.T) {
 
 func TestFlowAggregator_Run(t *testing.T) {
 	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
```

Comment:
it's no longer needed to explicitly call `ctrl.Finish` with recent versions of the gomock library

Reply:
Thanks, will update this along with other usages in a follow up PR.